docs/podman-run.1.md

```diff
@@ -214,6 +214,13 @@ inside of the container.
    Expose a port, or a range of ports (e.g. --expose=3300-3310) to set up port redirection
    on the host system.
 
+**--gidmap**=map
```

Comment:
Basically the same changes I requested for the 'create' manpage for this file, please.

We really need a way to share the content of these files...